laserscan_kinect/include/laserscan_kinect/laserscan_kinect.h

```diff
@@ -242,6 +279,8 @@ class LaserScanKinect
     bool  ground_remove_enable_{false};     ///< Determines if remove ground from output scan
     float ground_margin_{0};                ///< Margin for floor remove feature (in meters)
     bool  tilt_compensation_enable_{false}; ///< Determines if tilt compensation feature is on
+    bool  slope_detection_{false};           ///< Determines if a slope should be removed from scan
```

Comment:
remove one space before the comment to align with comments above and below this line